BENCHMARKING.md

```diff
@@ -58,19 +58,71 @@ and on the Hadoop cluster (with a varying number of executors, as set by the `$E
-        --conf spark.default.parallelism=$(( 2 * $EXEC * 5 )) \
+        --conf spark.hadoop.mapreduce.input.fileinputformat.split.minsize=$(( $SPLIT_MB * 1024 * 1024 )) \
+        --conf spark.default.parallelism=$(( $EXEC * 5 )) \
         --conf spark.dynamicAllocation.enabled=false \
+        --conf spark.executor.extraJavaOptions="-XX:+UseG1GC -XX:InitiatingHeapOccupancyPercent=35" \
         -- normalize_and_discover \
-        -min_partitions $(( 2 * $EXEC * 5 )) \
+        -min_partitions $(( $EXEC * 5 )) \
         -exclude_targets hdfs://path/to/20130108.exome.targets.exclude.txt \
         -min_some_quality 29.5 \
         -print_metrics \
         -I "hdfs://path/to/DATA.2535.RD.txt" \
         -o "hdfs://path/to/DATA.2535.gff3"
 ```
 
-The `20130108.exome.targets.exclude.txt` is the concatenation
+The specific command when running DECA on AWS EMR on a cluster of 4 8-core i3.2xlarge nodes is below.
+
+```
+deca-submit \
+    --master yarn \
+    --deploy-mode cluster \
+    --num-executors 7 \
+    --executor-memory 22G \
+    --executor-cores 4 \
+    --driver-memory 22G \
+    --driver-cores 5 \
+    --conf spark.driver.maxResultSize=0 \
+    --conf spark.kryo.registrationRequired=true \
+    --conf spark.dynamicAllocation.enabled=false \
+    --conf spark.hadoop.mapreduce.input.fileinputformat.split.minsize=$(( 104 * 1024 * 1024 )) \
+    --conf spark.default.parallelism=28 \
+    --conf spark.executor.extraJavaOptions="-XX:+UseG1GC -XX:InitiatingHeapOccupancyPercent=35" \
+    -- normalize_and_discover \
+    -min_partitions 28 \
+    -exclude_targets "s3a://path/to/20130108.exome.targets.exclude.txt" \
+    -min_some_quality 29.5 \
+    -print_metrics \
+    -I "s3a://path/to/DATA.2535.RD.txt" \
+    -o "s3a://path/to/DATA.2535.RD.gff3" \
+    -multi_file
+```
+
+The specific command when running DECA on Databricks is below.
+
+```json
+[
+    "--class", "org.bdgenomics.deca.cli.DecaMain",
+    "--conf", "spark.serializer=org.apache.spark.serializer.KryoSerializer"
+    "--conf", "spark.kryo.registrator=org.bdgenomics.deca.serialization.DECAKryoRegistrator",
+    "--conf", "spark.kryo.registrationRequired=true",
+    "--conf", "spark.hadoop.fs.s3.impl=com.databricks.s3a.S3AFileSystem",
+    "--conf", "spark.hadoop.fs.s3a.impl=com.databricks.s3a.S3AFileSystem",
+    "--conf", "spark.hadoop.fs.s3n.impl=com.databricks.s3a.S3AFileSystem",
+    "--conf", "spark.hadoop.fs.s3a.canned.acl=BucketOwnerFullControl",
+    "--conf", "spark.hadoop.fs.s3a.acl.default=BucketOwnerFullControl",
+    "--conf", "spark.dynamicAllocation.enabled=true",
+    "s3://databricks-deca/deca-cli_2.11-0.2.1-SNAPSHOT.jar",
+    "normalize_and_discover",
+    "-exclude_targets", "s3a://path/to/20130108.exome.targets.exclude.txt",
+    "-min_some_quality", "29.5",
+    "-I", "s3a://path/to/DATA.2535.RD.txt",
+    "-o", "s3a://path/to/DATA.2535.RD.gff3",
+    "-multi_file"
+]
+```
+
+The `20130108.exome.targets.exclude.txt` file is the concatenation
```

Comment:
`the concatenation` &rarr; `the concatenation of`

Reply:
Fixed.